modules/addons/ratelimit/main.tf

```diff
@@ -0,0 +1,41 @@
+provider "kubectl" {
+  host                   = var.k8s_host
+  cluster_ca_certificate = base64decode(var.k8s_cluster_ca_certificate)
+  token                  = var.k8s_client_token
+  load_config_file       = false
+}
+
+data "external" "redis_image" {
```

Comment:
Using this to fetch the image list and always set the right version of the redis image for the configured TSB version so we don't need to hardcode it and keep updating it.